website/docs/API.md

```diff
@@ -251,13 +252,31 @@ toJSON(): ReactTestRendererJSON | null
 
 Get the rendered component JSON representation, e.g. for snapshot testing.
 
-### `container`
+### `root`
 
 ```ts
-container: ReactTestInstance;
+root: ReactTestInstance;
 ```
 
-A reference to the rendered root element.
+Returns the rendered root [host element](testing-env#host-and-composite-components).
+
+This API is primarily useful in component tests, as it allows you to access root host view without using `*ByTestId` queries or similar methods.
```

Comment:
I feel like it could be interesting to give more indications on what we think are good use cases for this api (e.g. asserting an element is not rendered) because I don't think it's very intuitive how to use this